salt/states/pkg.py

```diff
@@ -508,8 +508,11 @@ def _find_install_targets(name=None,
         # add it to the kwargs.
         kwargs['refresh'] = refresh
 
+
+    resolve_capabilities = kwargs.get('resolve_capabilities', False) and 'pkg.list_provides' in __salt__
     try:
         cur_pkgs = __salt__['pkg.list_pkgs'](versions_as_list=True, **kwargs)
+        cur_prov = resolve_capabilities and __salt__['pkg.list_provides'](**kwargs) or dict()
```

Comment:
I would just call `list_provides(**kwargs)`, passing whatever in `resolve_capabilities`. And it already returns `{}` anyway.

Reply:
We are here in the package state and not in the zypper module.
Here we do not have such a function.